Source/WebCore/page/LocalFrameView.h

```diff
@@ -191,6 +191,8 @@ class LocalFrameView final : public FrameView {
     // Returns true if the flush was completed.
     WEBCORE_EXPORT bool flushCompositingStateIncludingSubframes();
 
+    WEBCORE_EXPORT void enableRenderingSuppressionStateIncludingSubframes();
```

Comment:
I was confused about the "state" in the name, but now I realize this is "update rendering suppression based on the view transition state". Maybe it needs "View transition" in the name. Perhaps `updateRenderingSuppressionForViewTransitions()` or something.